dwave_networkx/drawing/chimera_layout.py

```diff
@@ -230,87 +230,88 @@ def draw_chimera(G, **kwargs):
 
 
 def draw_chimera_embedding(G, *args, **kwargs):
-    """Draws an embedding onto the chimera graph G, according to layout.
+    """Draw an embedding onto the chimera graph ``G``, according to the Chimera layout.
```

Comment:
What does ", according to the Chimera layout" add to comprehension here?

Reply:
Are you saying that we can delete this phrase because it is in the Chimera layout by context?
Also, can we delete "in a Chimera cross topology" in other places. Also, what is a "cross topology"?

Reply:
Yes, I don't understand what the "according to layout" clause adds to the preceding clause here. Perhaps there's good intention but insufficiently clear writing behind the existing sentence.  

I can't answer the general question of deleting something in other places without knowing what those places say. 

For "cross" you might be referring to [cross](https://docs.dwavesys.com/docs/latest/c_gs_4.html#chimeraunitcell) layout drawing as opposed to column layout. Whether or not that info is essential or not, depends on the local context.